mobile/src/lib/localStorage.js

```diff
@@ -0,0 +1,21 @@
+import localforage from 'localforage'
+
+export const loadState = async () => {
+  try {
+    const persistedState = await localforage.getItem('state')
+    if (persistedState === null) {
+      return undefined
+    }
+    return persistedState
+  } catch (err) {
+    return undefined
```

Comment:
I don't know what error could append here, so I mute it and just let the redux store be initialized without any previous persisted state by returning `undefined`.
Nevertheless I guess it could help if we add a `console.warn` in the catch block.